src/parser/jobs/whm/modules/Overheal.tsx

```diff
@@ -16,6 +16,7 @@ export class Overheal extends CoreOverheal {
 				this.data.statuses.REGEN.id,
 				this.data.actions.MEDICA_II.id,
```

Comment:
Just noticed - Can you add the status for Medica_II aswell?

Reply:
Fixed, guess I missed that oops~